lib/pybind11/include/pybind11/detail/internals.h

```diff
@@ -191,7 +191,9 @@ PYBIND11_NOINLINE inline internals &get_internals() {
         auto *&internals_ptr = *internals_pp;
         internals_ptr = new internals();
 #if defined(WITH_THREAD)
-        PyEval_InitThreads();
+        #if PY_VERSION_HEX < 0x03090000
```

Comment:
What is this code doing?

Reply:
Short story: I do not know.
Long story: According to docs.python.org "In Python 3.6 and older, this function created the GIL if it didn’t exist". In Python 3.9 the `PyEval_InitThreads()` command is deprecated and it will be completely removed by Python 3.11. This code is checking the version of Python. If it is less than 3.9, `PyEval_InitThreads()` is executed, otherwise it is not. By putting this if statement I could compile QForte using Python 3.9 without any deprecation warning messages.

Reply:
Do _not_ edit the code in `lib`.

Those are external libraries used by QForte - we shouldn't change them. What QForte currently does (including its own copy of those libraries) is bad practice. Making this disagree from the standard versions is worse practice.

Reply:
I do understand that this is an external library. The change that I made is pretty innocent, but I will revert back. As you imply, we shouldn't be providing our own copy, because codes get updated. In a future PR, we should fix this.